utils/sb2.sh.in

```diff
@@ -1152,11 +1152,6 @@ END
 -- CPU transparency settings. Automatically generated file, do not edit.
 END
 
-    for liblocation in $sbox_liblocations ; do
-        if [ -d $liblocation ] ; then
-            break
-        fi
-    done
 	library_interface=$(LD_LIBRARY_PATH=$liblocation:$LD_LIBRARY_PATH $SBOX_BINDIR/sb2-show libraryinterface)
```

Comment:
The code is used in line 1160 below.

Reply:
Apparently `$liblocation` leaks out of the loop... I'll drop this commit.

Reply:
That is on purpose, in bash you can use variables which in other languages would belong to the scope of the loop.